samples/matter/lock/src/credentials/credentials_manager_credentials.cpp

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "credentials_manager.h"
+#include "credentials_storage.h"
+
+#include <platform/CHIPDeviceLayer.h>
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(cr_manager, CONFIG_CHIP_APP_LOG_LEVEL);
+
+using namespace chip;
+using namespace DoorLockData;
+
```

Comment:
Are the `credentials_manager_x.cpp` files added to the compilation anywhere?

Reply:
Btw. this is what I meant before about renaming the modules. Credentials manager credentials sounds weird.

Reply:
So as I wrote above, it can be transformed to "lock_persistent" for example.